config/local.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
Why was this file modified?

Reply:
I was attempting to get my local dev working. I'll revert the change.